api/v1/flow/flow.proto

```diff
@@ -521,7 +542,7 @@ message FlowFilter {
     // only return Flows that were classified with a particular verdict.
     repeated Verdict verdict = 5;
     // only applicable to Verdict = DROPPED (e.g. "POLICY_DENIED", "UNSUPPORTED_L3_PROTOCOL")
-    repeated DropReason drop_reason_desc = 35;
+    repeated DropReason drop_reason_desc = 33;
```

Comment:
Isn't this a breaking change?

Reply:
Strictly speaking yes, it is a breaking change. In practice there are no Cilium release yet with this field, so it should be safe to go. See [this tread](https://github.com/cilium/cilium/pull/32135#discussion_r1580739954) on the PR introducing the `drop_reason_desc` field and the commit message:
> Both fields were introduced recently (i.e. after v1.15) and thus this
commit doesn't break backward compatibility with any currently released
Cilium version at the time of writting.

Reply:
Oh right; I forgot this was new in the filters.